test/test9/test.sh

```diff
@@ -14,6 +14,11 @@
 set -e
 set -x
 cd `dirname $0`/../..
-ocamlc -I src ocamlbuild_pack.cmo test/test9/testglob.ml -o ./testglob.native
+if [ "$OCB" = "ocamlbuild" ]; then
+    IDIR="-I $($OCB -where | tr -d '\r')";
```

Comment:
to prevent issues for local runs in case cwd contains characters that are parsed differently by sh
```suggestion
    IDIR="-I '$($OCB -where | tr -d '\r')'";
```